packages/3id-did-resolver/package.json

```diff
@@ -9,7 +9,14 @@
     "Data"
   ],
   "author": "Joel Torstensson <oed3933@gmail.com>",
-  "homepage": "",
+    "repository": {
```

Comment:
please fix the formatting/indentation throughout all the files here

Reply:
thank you! I'll fix this asap

Reply:
fixed